doc/TUTORIAL_SCRIPT.md

```diff
@@ -0,0 +1,51 @@
+# Script
+
+```shell script
+docker run -it elektra/elektra                                      > NEW LINE
+
+sudo kdb mount hello.js /hello json
+kdb set user/hello/hello/world "Hello World"
+kdb get user/hello/hello/world
+kdb file user/hello/hello/world
+vi /home/elektra/.config/hello.js
+# Edit file to say "Hello World from File!"
+kdb get user/hello/hello/world
+kdb editor user/hello ini
+# Edit file to say "Hello World from INI!"
+kdb get user/hello/hello/world
+echo "Elektra can export to various formats!"
+kdb export user/hello
+kdb export user/hello tcl
+kdb export user/hello yaml                                      > NEW LINE
+echo "Remove entries and file"
+kdb rm user/hello/hello/world
+cat /home/elektra/.config/hello.ini
+sudo kdb umount /hello
+echo "Now let us change something.. With validation"
+vi specification.ini                                       > NEW START
+# content:
+# []
+# mountpoint = specification.ini
+# [hostip]
+# check/ipaddr =
+# [timeout]
```

Comment:
Again: I think this is too much at once, build this up one by one (simply call spec-mount again whenever you add new meta-data to the file)